app/models/version.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# The Version class represents the version model
+class Version < ApplicationRecord
+  has_many :screenshots
+  belongs_to :products
```

Comment:
`belongs_to` must be singular

For example: [rails guide](https://guides.rubyonrails.org/association_basics.html#the-belongs-to-association)

`belongs_to :product`